app/code/Magento/User/Test/Unit/Controller/Adminhtml/Auth/ForgotPasswordTest.php

```diff
@@ -0,0 +1,322 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\User\Test\Unit\Controller\Adminhtml\Auth;
+
+use Magento\Backend\App\Action\Context;
+use Magento\Backend\Helper\Data;
+use Magento\Framework\App\Request\Http;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\App\ViewInterface;
+use Magento\Framework\Controller\Result\Redirect;
+use Magento\Framework\Controller\Result\RedirectFactory;
+use Magento\Framework\Exception\SecurityViolationException;
+use Magento\Framework\Message\ManagerInterface;
+use Magento\Security\Model\PasswordResetRequestEvent;
+use Magento\Security\Model\SecurityManager;
+use Magento\User\Controller\Adminhtml\Auth\Forgotpassword;
+use Magento\User\Model\ResourceModel\User\CollectionFactory;
+use Magento\User\Model\Spi\NotificatorInterface;
+use Magento\User\Model\User;
+use Magento\User\Model\UserFactory;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ */
+class ForgotPasswordTest extends TestCase
+{
+    /**
+     * @var Data|MockObject
+     */
+    private $backendDataHelper;
+
+    /**
+     * @var Forgotpassword
+     */
+    private $controller;
+
+    /**
+     * @var ManagerInterface|MockObject
+     */
+    private $messageManager;
+
+    /**
+     * @var Http|MockObject
+     */
+    private $request;
+
+    /**
+     * @var ResponseInterface|MockObject
+     */
+    private $response;
```

Comment:
Could you add `Mock` suffix to all the mocks, like this?
```suggestion
    private $responseMock;
```